docs/reference/samples/observability.output-es-managed.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: "observability.openshift.io/v1"
```

Suggest to change to unquoted:

```suggestion
apiVersion: observability.openshift.io/v1
```